editor/js/ui/workspaces.js

```diff
@@ -240,18 +241,20 @@ RED.workspaces = (function() {
                 }
             },
             onadd: function(tab) {
+                workspaceTabCount = tab.type === "tab"?workspaceTabCount+1:workspaceTabCount;
```

Comment:
From a code style point of view, I'd prefer if this was the easier to read alternative:
```
if (tab.type === "tab") {
   workspaceTabCount++;
}
```